internal/cli/status.go

```diff
@@ -17,18 +16,20 @@ import (
 	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
 	"github.com/hashicorp/waypoint/internal/clicontext"
 	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/clijob"
 	"github.com/hashicorp/waypoint/internal/pkg/flag"
 	pb "github.com/hashicorp/waypoint/internal/server/gen"
+	"github.com/hashicorp/waypoint/internal/version"
 )
 
 type StatusCommand struct {
 	*baseCommand
 
-	flagContextName string
-	flagVerbose     bool
-	flagJson        bool
-	flagAllProjects bool
-	filterFlags     filterFlags
```

Comment:
Did you mean to remove this?

Reply:
Yep, in the original implementation I thought about using filter flags with what's built in, but it turns out those flags are about filtering on the operations status. I removed the flag definition but never removed the bool from the struct!